src/docs/wtperf.dox

```diff
@@ -195,7 +195,7 @@ close and reopen the connection between populate and workload phases
 @par report_interval (unsigned int, default=2)
 output throughput information every interval seconds, 0 to disable
 @par run_ops (unsigned int, default=0)
-total read, insert and update workload operations
+total insert, modify, insert and update workload operations
```

Comment:
This is generated, so this doc should be fixed when the above `.i` file is fixed.

Reply:
Fixed.